### ISSUE TITLE
```inst_jns.sh``` stops during TeX installation

### ISSUE THREAD

**Issue by kleinee** [kleinee/jns#38]
On two fresh installations (3B+ // ZeroW ) ```inst_jns.sh``` stopped during execution of ``inst_tex.sh``` for no apparent reasons.  Starting ```inst_tex.sh``` manually afterwords worked without problems. Please comment here if you experienced the same issue know how to fix.

**Comment by kleinee** [comment]
```bash
...
Setting up rake (10.5.0-2) ...
Setting up libnet-dbus-perl (1.1.0-4+b1) ...
Processing triggers for libc-bin (2.24-11+deb9u4) ...

[1]+  Stopped                 sudo ./inst_jns.sh
```
Just to take note where the process was stopped. Don't know why the execution got interrupted but it reading up on it online I should have been able to continue with ```fg```

**Comment by kleinee** [comment]
```bash
...
Setting up rake (10.5.0-2) ...
Setting up libnet-dbus-perl (1.1.0-4+b1) ...
Processing triggers for libc-bin (2.24-11+deb9u4) ...

[1]+  Stopped                 sudo ./inst_jns.sh
pi@raspberrypi:~/jns/scripts $ fg
sudo ./inst_jns.sh
Reading package lists... Done
Building dependency tree 
...
```

The script still stops at the same point - `fg` resumes installation. Quite odd as this did not happen in the past. All I had running at the same time was `top` in a second terminal window.

**Comment by kleinee** [comment]
As I currently do not have the time to hunt for a fix I suggest to rune the scripts by hand in the order as outlined in `README.md`. Note that the Julia 1.1.0 installer either needs to be tweaked or the the download helper needs to be placed in /home/pi.

**Comment by qqgg231** [comment]
I can't reproduce this bug on clean install of Raspbian Buster on 3A+ and 3B

**Comment by kleinee** [comment]
Hi @qqgg231,

Good news as far as this issue goes. Buster was just released and now ships Python 3.7.  The number of 3.7 packages on [piwheels](https://www.piwheels.org/) is still limited. What it is taking time is compilation. This will significantly improve as more packages become available. Thanks for your endurance - your are probably among the very first trying the scripts on Buster - Thanks.

Have Fun!

Eckhard
